pkg/addon/configpolicy/manifests/managedclusterchart/templates/monitoring_role.yaml

```diff
@@ -18,6 +18,9 @@ rules:
     resources:
       - services
       - pods
+      - endpoints
+      - nodes
```

Comment:
Seems odd that nodes are required here. I can kind of understand endpoints and maybe secrets though... But this is what's doc'd so, it seems good to me!